spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/integration/IntegrationProperties.java

```diff
@@ -295,4 +302,86 @@ public void setMessageMappingEnabled(boolean messageMappingEnabled) {
 
 	}
 
+	public static class Poller {
+
+		/**
+		 * Maximum of messages to poll per polling cycle.
+		 */
+		private Long maxMessagesPerPoll;
+
+		/**
+		 * How long to wait for messages on poll.
+		 */
+		private Duration receiveTimeout;
+
+		/**
+		 * Polling delay period.
+		 */
+		private Duration fixedDelay;
+
+		/**
+		 * Polling rate period.
+		 */
+		private Duration fixedRate;
+
+		/**
+		 * Polling initial delay.
```

Comment:
It would be good to document that `cron`, `fixedRate`, and `fixedDelay` are mutually exclusive.